homeassistant/components/transmission/translations/en.json

```diff
@@ -23,6 +23,7 @@
                     "name": "Name",
                     "password": "Password",
                     "port": "Port",
+                    "ssl": "Uses an SSL certificate",
```

Comment:
The translation files should no longer be included in the PR.
Please adjust accordingly.